src/transformers/models/kosmos2/processing_kosmos2.py

```diff
@@ -145,10 +157,27 @@ def __call__(
         if images is None and text is None:
             raise ValueError("You have to specify either images or text.")
 
+        bboxes = kwargs.pop("bboxes", None)
+        num_image_tokens = kwargs.pop("num_image_tokens", 64)
+        first_image_token_id = kwargs.pop("first_image_token_id", None)
+        add_eos_token = kwargs.pop("add_eos_token", False)
```

Comment:
Not sure if this is the correct way to deal with model-specific extra kwargs. I guess the whole point of uniformizing the kwargs is to not have something like that, but I don't know how to deal with these very specific kwargs

Reply:
Oh I see, that's nice!

Reply:
yes I think it's fine to use `pop` or `get` in this case :)